deployment/terraform/database.tf

```diff
@@ -100,5 +98,5 @@ module "database" {
   insufficient_data_actions          = [aws_sns_topic.global.arn]
 
   project     = var.project
-  environment = var.environment
+  environment = ""
```

Comment:
Our RDS module will set this to `Unknown` if omitted, which felt like it would look unintentional and confusing. I set this to an empty string here so any templated names concatenate and empty string and end up cleanly omitting the environment. The `Environment` tag was still created, but it's set to an empty string.

Reply:
This approach is fine. An imperfect alternative could be to keep the project level `environment` variable and use `Production` as its value, but just not interpolate it into resource names.